### DISCUSSION TITLE
is it possible to not send the <script> tag over ?

### DISCUSSION THREAD

**Discussion by G3z** [livewire/livewire#3791] [1. Help]
in my script tag I do print some php data to json (like 3mb of it)
i've noticed the script tag is sent over the wire on every update but that content is useless 
(<script> tag is ignored after the first render)
is it possible to ignore it when sending data `wire:ignore` does not work? 
maybe it's also possible to skip rendering of <script> tags after the first render

**Comment by joshhanley** [comment]
@G3z one way would be to use `@push('scripts')` and push the script tag to your layout file, as that only gets sent on the first render.

Hope this helps!

**Reply by G3z** [reply]
Nice, thank you! 
I suggest adding a note here https://laravel-livewire.com/docs/2.x/inline-scripts

**Reply by joshhanley** [reply]
@G3z no worries 🙂 PR's are always welcome for the docs at https://github.com/livewire/docs
